tests/functional/verification/issues/00096.py

```diff
@@ -0,0 +1,26 @@
+from nagini_contracts.contracts import *
+from nagini_contracts.obligations import MustTerminate
+from nagini_contracts.thread import Thread
+
+
+class Cell:
+
+     def __init__(self) -> None:
+         Ensures(Acc(self.x))
+         self.x = 0  # type: int
+
+     def void(self) -> None:
+         Requires(Acc(self.x, 1/4))
+         Requires(MustTerminate(2))
+         Ensures(Acc(self.x, 1/4))
+
+     def a12(self, a: int) -> None:
```

Comment:
Is there any verification error for the fact that parameter 'a' could be negative yielding an infinite loop?

Reply:
No, Nagini verifies partial correctness by default, so code is generally allowed to diverge. If you don't want that, you have to explicitly specify that things should terminate using a MustTerminate obligation.